### PR TITLE
fix quoting for non-tty case

### PR DESCRIPTION
**Pull request by mmoll**
closes GH-28

we need a new release after this.

### CODE REVIEW

**Comment by domcleal** [comment]
Is it possible to add a test for this?  Perhaps running `nohup freight-cache -c $FREIGHT_CONFIG` would work to disconnect the tty.

**Comment by mmoll** [comment]
indeed, that test fails without the fix.

File: test/apt_cache.bats [hunk 1]
```diff
@@ -40,6 +40,11 @@ setup() {
     gpg --verify ${FREIGHT_CACHE}/dists/example/Release.gpg ${FREIGHT_CACHE}/dists/example/Release
 }
 
+@test "freight-cache works without tty" {
+    freight_cache_nohup -v
+    assert_success
```
Comment (domcleal) [inline test/apt_cache.bats]: This only works with `run`, use that above.

Comment (domcleal) [inline test/apt_cache.bats]: Or just remove the line as there are no other assertions. The line above will exit on error.

**Comment by domcleal** [comment]
Thanks @mmoll!

**Pull request merged**